### DISCUSSION TITLE
post/javascript/keydown-keyup-twice

### DISCUSSION THREAD

**Discussion by giscus** [corinthionia/blog-comments#12] [Comments]
# post/javascript/keydown-keyup-twice

Notion처럼 텍스트를 입력하고 엔터를 누르면 새로운 블럭이 만들어지는데, 기존 텍스트의 마지막 음절이 딸려오는 에러가 발생했다.

https://corinthionia.github.io/post/javascript/keydown-keyup-twice

**Comment by Yunil-dev** [comment]
잘 보고 갑니다! 덕분에 쉽게 수정했어요✍️
